s3readerat.go

```diff
@@ -4,6 +4,7 @@ package s3readerat
 import (
 	"context"
 	"fmt"
+	awshttp "github.com/aws/aws-sdk-go-v2/aws/transport/http"
```

Comment:
this import should not be in this line since it is not part of the standard lib, please run `goimports` to fix the imports groups